sdcm/sct_config.py

```diff
@@ -1925,10 +1925,16 @@ def __init__(self):  # noqa: PLR0912, PLR0914, PLR0915
         # 9) append username or ami_id_db_scylla_desc to the user_prefix
         version_tag = self.get('ami_id_db_scylla_desc') or getpass.getuser()
         user_prefix = self.get('user_prefix') or getpass.getuser()
+        prefix_max_len = 35
```

Comment:
What about case when `version_tag == user_prefix:`?

Reply:
it does not match any `if` and going to be truncated the same (35) - in line 1937